### ISSUE TITLE
Method to pass user callback function?

### ISSUE THREAD

**Issue by fey** [voku/Stringy#28]
#### Any additional information?

Hello, there is exists method to pass user function, that modify string?
For example, in Laravel i have str_plural and class_basename so i could be use it in chain 
```php
s($myClass)
    ->callUserFunc(fn($myClass) => class_basename($myClass))
    ->toLowerCase()
    ->callUserFunc(fn($myClass) => str_plural($myClass));
```
or
```php
s($myClass)
    ->callUserFunc('class_basename')
    ->toLowerCase()
    ->callUserFunc('str_plural');
```

**Comment by voku** [comment]
Hi, good idea, I added this -> https://github.com/voku/Stringy/commit/989391ffd42ef073dfa4b4ff0433f4865796e1a5#diff-72c90db4478e945428a2eb3bcc72971dR843 <- any further ideas for this method, before I release this new feature? 

Bests
Lars

**Comment by fey** [comment]
Great! Perhaps someone will come up with a more convenient method name. Need to think :slightly_smiling_face:

**Comment by voku** [comment]
The new method is available in the current version >= 6.3.0
